ui/webui/src/components/review/ReviewConfiguration.jsx

```diff
@@ -25,26 +25,55 @@ import {
     DataListCell,
     DescriptionList, DescriptionListGroup,
     DescriptionListTerm, DescriptionListDescription,
+    ExpandableSection,
     Modal, ModalVariant,
     Alert
 } from "@patternfly/react-core";
 
 import {
-    getSelectedDisks, getDeviceData,
+    getSelectedDisks,
+    getDeviceData,
+    getAppliedPartitioning,
+    getPartitioningRequest,
+    getInitializationMode,
 } from "../../apis/storage.js";
 
 import {
     getLanguage, getLanguageData,
 } from "../../apis/localization.js";
 import { AnacondaPage } from "../AnacondaPage.jsx";
 
+import { scenarioForInitializationMode, getScenario } from "../storage/StorageConfiguration.jsx";
+
 const _ = cockpit.gettext;
 
-export const ReviewConfiguration = ({ idPrefix }) => {
+export const ReviewDescriptionList = ({ children }) => {
+    return (
+        <DescriptionList
+          isHorizontal
+          horizontalTermWidthModifier={{
```

Comment:
I wonder if these should maybe somehow got to the CSS file if possible ?

Reply:
Why do you think it should go to a CSS file? Seems to me like customization of a component by setting its properties. Following this example: https://www.patternfly.org/v4/components/description-list/#horizontal-using-custom-term-width-modifier.